k-distribution/k-tutorial/1_basic/07_side_conditions/README.md

```diff
@@ -68,42 +68,42 @@ What this means, in practice, is that this rule has lower priority than other
 rules, and will only be tried to be applied after all the other,
 higher-priority rules have been tried and they have failed.
 
-For example, in the above exercise, we had to add a side condition to the rule
-we wrote to handle `isPositive` for negative numbers. However, in practice this
-meant that the integer comparison operation happened twice. We can more
-efficiently and more idiomatically write the negative case for the `isPositive`
-rule using the `owise` attribute (`lesson-07-b.k`):
+For example, in the above exercise, we had to add a side condition containing
+two boolean comparisons to the rule we wrote to handle `B()` grades. However,
+in practice this meant that we compare the percentile to 90 twice. We can more
+efficiently and more idiomatically write the `B()` case for the
+`gradeFromPercentile` rule using the `owise` attribute (`lesson-07-b.k`):
 
 ```k
 module LESSON-07-B
   imports BOOL
   imports INT
 
-  syntax Bool ::= isPositive(Int) [function]
+  syntax Grade ::= A() | B() | C() | D() | F()
+                 | gradeFromPercentile(Int) [function]
 
-  rule isPositive(I) => true  requires I >Int 0
-  rule isPositive(_) => false [owise] 
+  rule gradeFromPercentile(I) => A() requires I >=Int 90
+  rule gradeFromPercentile(I) => B() requires I >=Int 80 [owise]
 endmodule
 ```
 
-Note that we have introduced a new piece of syntax here: `_`. This is actually
-just a variable. However, as a special case, when a variable is named `_`, it
-does not bind a value that can be used on the right hand side of the rule, or
-in a side condition. Effectively, `_` is a placeholder variable that means "I
-don't care about this term."
-
-This rule is saying, "if the first rule does not apply, then the result of the
-function is false." Note, however, that `owise` can be used in more complicated
-ways. For example, you can perform additional matching in an `owise` rule. You
-can also have multiple higher-priority rules, or multiple `owise` rules. What
-this means in practice is that all of the non-`owise` rules are tried first, in
-any order, followed by all the `owise` rules, in any order.
+This rule is saying, "if the first rule does not apply, then the grade is a B
+if the percentile is greater than or equal to 80." Note here that we use both
+a side condition and an `owise` attribute on the same rule. This is not
+required (as we will see later), but it is allowed. What this means is that the
+side condition is only tried if the previous rules did not apply **and** the 
```

Comment:
```suggestion
side condition is only tried if the other rules did not apply **and** the 
```

I would prefer to say "other" instead of "previous" in these cases, because "previous" might imply that it means "earlier in the file", which we do not mean to imply.